src/core/Slider/Slider.tsx

```diff
@@ -12,6 +12,47 @@ import { getBoundedValue } from '../utils/common';
 import { Track } from './Track';
 import { Thumb } from './Thumb';
 
+// based on https://usehooks.com/useEventListener/
+function usePointerEventListener(
```

Comment:
This could be a generic hook under `utils/`. There is no PointerEvent-specific logic here.

Reply:
I did move it to utils/hook/usePointerEventListener,  but the handler type is 'PointerEvent-specific'

Reply:
Yeah I'm saying make it a generic `useEventListener`

Reply:
I should have done this initially... done now.